src/hotspot/share/runtime/os.cpp

```diff
@@ -518,6 +518,11 @@ void* os::native_java_library() {
     char buffer[JVM_MAXPATHLEN];
     char ebuf[1024];
 
+    if (vm_is_statically_linked()) {
```

Comment:
This block can be moved before the two variable declarations above, since they are not needed in the static case. https://github.com/openjdk/leyden/blob/c1c5fc686c1452550e1b3663a320fba652248505/src/hotspot/share/runtime/os.cpp#L507 handles it that way.

Reply:
Fixed.